src/lib/stitching/vortex/stitching.ts

```diff
@@ -282,29 +282,34 @@ export const vortexStitchingEnvironment = (localSchema: GraphQLSchema) => ({
               __typename
               ... on AnalyticsArtwork {
                 entityId
+                partnerId
               }
-              ... on AnalyticsShow {
+              ... on AnalyticsPartnerShow {
                 entityId
+                partnerId
               }
               ... on AnalyticsArtist {
                 entityId
+                partnerId
               }
             }
           }
         `,
         resolve: (parent, _args, context, info) => {
           const removeVortexPrefix = name => name.replace("Analytics", "")
           const typename = parent.rankedEntity.__typename
-          const fieldName = removeVortexPrefix(typename).toLowerCase()
+          const fieldName = snakeCase(removeVortexPrefix(typename))
```

Comment:
If I’m reading this right this is adding a snake case field? All new MP fields should be camelCase.